Framework/PythonInterface/plugins/algorithms/RefLReduction.py

```diff
@@ -76,7 +77,7 @@ def PyInit(self):
     #pylint: disable=too-many-locals, too-many-branches
     def PyExec(self):
 
-        print '-- > starting new Reflectometer Reduction ...'
+        print('-- > starting new Reflectometer Reduction ...')
```

Comment:
What log-level do you suggest to replace `print` in general? This looks like `info` or something, but sometimes it can be more like `debug` I suppose.